packages/tsx/find-esbuild-config.mjs

```diff
@@ -1,19 +1,19 @@
 import { createRequire, findPackageJSON } from 'node:module';
-import process from 'node:process';
 import { fileURLToPath } from 'node:url';
 
-
 // This config must contain options that are compatible with esbuild's `transform` API.
 let esbuildConfig;
 
 /**
  * @param {URL['href']} parentURL
  */
-export function findEsbuildConfig(parentURL) {
-  if (esbuildConfig != null) return esbuildConfig;
+export function findEsbuildConfig(parentURL){
+  if (esbuildConfig != null) return esbuildConfig
+
+	console.log('parentURL', parentURL)
 
-  const esBuildConfigLocus = findPackageJSON(parentURL)
-    ?.replace('package.json', 'esbuild.config.mjs');
+	const esBuildConfigLocus = findPackageJSON(parentURL, import.meta.url)
```

Comment:
I think the second argument is not what you want: that will make the search always relative to this file (`find-esbuild-config.mjs`). `parentURL` will always be fully qualified, so I think the second arg is ignored (there's nothing to resolve).

Reply:
i had add that to trying to fix issue with `findPacakgeJSON`